ci/build_wheel.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_name="cucim"
+package_dir="python/cucim"
+package_src_dir="${package_dir}/src/${package_name}"
+
+CMAKE_BUILD_TYPE="release"
+
+source rapids-configure-sccache
+source rapids-date-string
+
+version=$(rapids-generate-version)
+commit=$(git rev-parse HEAD)
+
+# for CMake VERSION_CPP need to truncate any trailing 'a'
+version_cpp=${version%a*}
+
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+
+# Patch project metadata files to include the CUDA version suffix and version override.
+pyproject_file="${package_dir}/pyproject.toml"
+
+PACKAGE_CUDA_SUFFIX="-${RAPIDS_PY_CUDA_SUFFIX}"
+# update package name to have the cuda suffix
+sed -i "s/name = \"${package_name}\"/name = \"${package_name}${PACKAGE_CUDA_SUFFIX}\"/g" ${pyproject_file}
+echo "${version}" > VERSION
+echo "${version_cpp}" > VERSION_CPP
+echo "${version_cpp}" > "${package_dir}/VERSION"
+sed -i "/^__git_commit__/ s/= .*/= \"${commit}\"/g" "${package_src_dir}/_version.py"
+
+if [[ ${PACKAGE_CUDA_SUFFIX} == "-cu12" ]]; then
+    # change pyproject.toml to use CUDA 12.x version of cupy
+    sed -i "s/cupy-cuda11x/cupy-cuda12x/g" ${pyproject_file}
+fi
+
+echo "The package name and/or version was modified in the package source. The git diff is:"
+git diff
```

Comment:
note: JFYI we've removed this from every other repo at this point but you're welcome to keep it if you find it useful for debugging. It's been a long time since we got much use out of it.

Reply:
Okay, I can remove it